src/rules/context-close.test.ts

```diff
@@ -0,0 +1,103 @@
+import rule from '../../src/rules/context-close'
+import { javascript, runRuleTester } from '../utils/rule-tester'
+
+runRuleTester('context-close', rule, {
+  invalid: [
+    {
+      code: javascript`
+            function createContext() {
+            
+            }
+    
+            function close() {
+            
+            }
+    
+    
+            (() => {
+                createContext();
+            })()
```

Comment:
I don't love how broad this rule is. IMO, this should only apply to `browser.createContext`, but that said, I'm not a huge fan of the rule in the first place, given how niche it is to create a new context, so maintaining another rule just for that seems unecessary.

Reply:
is possibile to apply it to `browser.createContext` only if necessary, but I could see a lot of usage coming from abstraction of this.

For how niche it is, meh. Really every usage of playwright with multiple users are going to use `context` if they want  appropriate coverage with different behaviour of users with different roles or authenticated vs non authenticated. Happy to refactor in the scope of `browser.createContext` only or close the PR completely if deemed unnecessary.

Reply:
If you are doing multi-user tests, I would highly recommend creating a fixture (e.g. `context2` and then have the fixture do the work of opening/closing the context.

Not only does this remove the need for the rule, but it moves the boilerplate and possibility of error out of your tests into a single location in your fixtures. Much easier to manage, way easier to write tests.

```
test('', ({ context, context2 }) => {
  // ...
})
```

Reply:
It's really not scalable due to multi user/multi page interactions, would be fesible if playwright allow the extend to be a factory function that could generate the necessary context, but is not at the minute. Whatever, I will close the Pr if not interested.